pom.xml

```diff
@@ -99,7 +99,13 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>git</artifactId>
-            <version>2.4.0</version>
+            <version>2.4.0</version> <!-- should really be 2.6.4 to match scm-api but ok as git wass forward compat -->
```

Comment:
Workflow tests need rework for the updated git dependency, but as git plugin didn't break the rules of SCM API we don't *need* to update to 2.6.4 / 3.0.4... just would be missing the SCMEvent integration in git plugin, but the tests in this plugin do not rely on that functionality, so no harm